versioned_docs/version-2.8/api/previous-rancher-api-versions/previous-rancher-api-versions.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Previous API Guide
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/api/previous-rancher-api-versions"/>
+</head>
+
+## How to use the API
+
+The previous Rancher API has its own user interface accessible from a web browser.  This is an easy way to see resources, perform actions, and see the equivalent cURL or HTTP request & response.  To access it:
+
+<Tabs>
+<TabItem value="Rancher v2.6.4+">
```

Comment:
Is this still accessible in Rancher 2.8.x? If not, it might be preferable to link to https://ranchermanager.docs.rancher.com/v2.7/reference-guides/about-the-api

Reply:
This is not accessible as of 2.8, went with your suggestion and added in a link.

Reply:
This is still available in 2.8. It's the `### Enable View in API` section found around line 86 later in this file.

Reply:
Hey Billy thanks for shouting this out! I think I got a bit confused when initially creating this PR so with your added clarification and this [comment](https://github.com/rancher/rancher-docs/issues/1089#issuecomment-1989476517) from the parent issue I am adjusting the document content for both pages and their placement in an upcoming commit. Just wanted to note it here.